manual/guide-tweeting.md

```diff
@@ -6,9 +6,9 @@ path: /how-to-tweet
 tags: ["guidelines", "community-manual"]
 ---
 
-Thanks so much for volunteering your time and energy! This is a handy, at-a-glance guide to livetweeting a DMG event, including workshops, Speaker’s Socials, or showcases.
+Thanks so much for volunteering your time and energy! This is a handy, at-a-glance guide to livetweeting a DMG event, including workshops, Speakers Socials, or showcases.
```

Comment:
@henryfaber I would lower-case and make singular "speaker socials" here

Reply:
Got it! Will amend.